src/test/java/com/woowacourse/matzip/domain/campus/CampusTest.java

```diff
@@ -0,0 +1,20 @@
+package com.woowacourse.matzip.domain.campus;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class CampusTest {
```

Comment:
```suggestion
public class CampusTest {
```

다른 테스트들과의 컨벤션을 맞추는게 좋을거같아영